web/src/app/modules/shared/components/presentation/datagrid/datagrid.component.html

```diff
@@ -46,9 +46,16 @@ <h4 class="clr-col-10" *ngIf="showTitle()">{{ title }}</h4>
     <clr-dg-cell *ngFor="let column of columns; trackBy: identifyColumn">
       <app-view-container [view]="row.data[column]"></app-view-container>
     </clr-dg-cell>
-    <ng-container ngProjectAs="clr-dg-row-detail" *ngIf="row?.expandedDetail">
+    <ng-container ngProjectAs="clr-dg-row-detail" *ngIf="row?.expandedDetails">
       <clr-dg-row-detail *clrIfExpanded [clrDgReplace]="row?.replace">
-        <app-view-container [view]="row.expandedDetail"></app-view-container>
+        <ng-template [ngIf]="row.expandedDetails.length > 1" [ngIfElse]="loading">
```

Comment:
Let's use a better name for the else template since it isn't really loading here.

Reply:
Done!